cmd/swctl/app/cmd_dependency.go

```diff
@@ -0,0 +1,482 @@
+package app
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"text/template"
+
+	"golang.org/x/exp/slices"
+
+	"github.com/gookit/color"
+	"github.com/olekukonko/tablewriter"
+	"github.com/spf13/cobra"
+)
+
+func exampleDependencyCmd(appName string) string {
+	return `
+  <white># Status of all dependencies</>
+  $ <yellow>` + appName + ` dependency status</>
+
+  <white># Install external tools (docker, docker compose)</>
+  $ <yellow>` + appName + ` dependency install-ext </>
+
+  <white># Set HugePages in kB manually to size 2048kB</>
+  $ <yellow>` + appName + ` dependency hugepages <value></>
+
+  <white># Assign(up) or Unassign(down) interfaces to/from kernel</>
```

Comment:
Added dpdk driver assigning

Reply:
After seeing the new implementation where it also binds/unbinds drivers, the text is right now (even it little bit hides also the state changes). The link up/down command is does also the driver changes that is not expected but the above text could explain it. 
Why did you change it to pci(long number) instead of interface names?

Reply:
Because pci (number) is doesn't change unlike interface name, which is able only when device is assigned to network kernel driver.
It could be good idea to predict interface name and use it but unfortunately I have zero expierence with it.

Reply:
So the reason is that we don't have it when pci network device is not handled by linux kernel by appropriate driver. That is reasonable. Ok, lets stick with PCI number for now. Trying to guess network interface naming assigned from kernel is messy even when you know the correct logic (i think there is longer if-elseif-elseif... tree of possibilities for the naming in kernel)